### DISCUSSION TITLE
VeraCrypt

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#79] [General]
# VeraCrypt

Check if VeraCrypt is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/011d762e-37f3-49b9-8d4a-0bc33210904a

**Comment by jinzapper** [comment]
No versions of the installer of Veracrypt works with Windows ARM natively.
